Practice tasks pull request

IvanTyurin opened the pull request:
- [x] Create GitHub account
- [x] Configure git clients (CLI, SourceTree)
- [x] Create Flutter project
- [x] Get a new branch practice from master
- [x] Add project to version control system
- [x] Add correct .gitignore Flutter Dart
- [x] Add lecturer to reviewers
- [x] Create draft Pull-request
- [x] Solve a problem
- [x] Update Pull-request
- [x] Add git tag "final solution"

IvanTyurin committed:
Remove some extra code

Deleted autogenerating code in main.dart

IvanTyurin committed:
UI

Create user interface

IvanTyurin committed:
The task is solved

The program expects the user to enter a number.
After input, it compares with a randomly generated one.
If the user guesses correctly, a congratulation appears and a start over button.
If more or less, then information about it is displayed.

vasilevkin commented on lib/main.dart:
```diff
@@ -1,3 +1,5 @@
+import 'dart:math';
```
Use Code> Reformat code with dartfmt , before every save.

IvanTyurin replied:
Done

vasilevkin commented on lib/main.dart:
```diff
@@ -9,109 +11,136 @@ class MyApp extends StatelessWidget {
   }
 }
 
 class MyHomePage extends StatefulWidget {
```
move to separate file

IvanTyurin replied:
Done

vasilevkin commented on lib/main.dart:
```diff
@@ -9,109 +11,136 @@ class MyApp extends StatelessWidget {
         visualDensity: VisualDensity.adaptivePlatformDensity,
       ),
-      home: MyHomePage(title: 'Flutter Demo Home Page'),
+      home: MyHomePage(title: 'Practice 2'),
```
Why title is in the constructor. It's never changes.

IvanTyurin replied:
Done

vasilevkin commented on lib/main.dart:
```diff
@@ -9,109 +11,136 @@ class MyApp extends StatelessWidget {
 
 class _MyHomePageState extends State<MyHomePage> {
-  int _counter = 0;
+  Random rnd = Random();
```
make understandable variable names

IvanTyurin replied:
Done

vasilevkin reviewed (changes requested):
request changes

IvanTyurin committed:
Comments fix

- [x] Removed the constructor. Title is now constant.
- [x] Created class HomeScreen, and moved HomePage there.
- [x] Renamed the variable.

vasilevkin committed:
Auto stash before merge of "dev/practice" and "origin/dev/practice"

IvanTyurin committed:
Merge branch 'somebranch' into dev/practice

# Conflicts:
#	lib/main.dart
